bin/archive_files.py

```diff
@@ -174,9 +177,8 @@
             # Now, push the new file
             # push the file to FIRE where fire_path will the path in the FIRE
             # filesystem
-            fireObj = api.push_object(fileO=f_in_staging,
-                                      dry=str2bool(args.dry),
-                                      fire_path=fire_path)
+            fireObj = api.upload_s3_object(firePath=f_in_staging,
```

Comment:
Hi @olaaustine, thank you for the changes. Do we want to handle api.upload_s3_object to return None as we check for return type before removing file: https://github.com/igsr/igsr_archive/blob/902179de06fc083908a27ae7a2d78e1b063b7b82/bin/archive_files.py#L220